fee refunds: document new fee refund by recipient RPCs (#582)
flashbots/flashbots-docs: docs/flashbots-auction/advanced/rpc-endpoint.mdx

- `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
- `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle

### flashbots_getFeeRefundTotalsByRecipient

> The `flashbots_getFeeRefundTotalsByRecipient` JSON-RPC method returns the total amount of fee refunds that have been processed for a specific address.

Reviewer: Can we make "fee refunds" a link to the page in the docs? https://docs.flashbots.net/flashbots-protect/gas-fee-refunds

Reviewer: What do you think of saying "earned by" instead of "processed"? I think this is more precise.

> The `flashbots_getFeeRefundTotalsByRecipient` JSON-RPC method returns the total amount of fee refunds that have been processed for a specific address.  Our refund process calculates these values weekly.

Reviewer: "processed for" -> "earned by"

> The `flashbots_getFeeRefundTotalsByRecipient` JSON-RPC method returns the total amount of fee refunds that have been processed for a specific address.  Our refund process calculates these values weekly.

Author: ```suggestion
The `flashbots_getFeeRefundTotalsByRecipient` JSON-RPC method returns the total amount of fee refunds that have been earned by a specific address.  Our refund process calculates these values weekly.
```

```json
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "flashbots_getFeeRefundTotalsByRecipient",
  "params": [
    {
      recipient,       // String, the address to query for fee refunds
    }
  ]
}
```

The response contains three fields:

```json
{
  "pending":"0x17812ea4fbbe314",
  "received":"0x108d1b27b63a213",
  "maxBlockNumber":"0x13ddb08"
}
```

> - `pending`: the total amount of fee refunds that have been processed but not yet sent to the recipient

Reviewer: Nit: would say "not yet received by the recipient" instead of "sent" to be forwards compatible with a pull-based refund model

> - `pending`: the total amount of fee refunds not yet received by the recipient

Reviewer: "the total amount of fee refunds that have been earned but not yet received by the recipient"

> - `pending`: the total amount of fee refunds not yet received by the recipient

Author: ```suggestion
- `pending`: the total amount of fee refunds that have been earned but not yet received by the recipient
```

> - `received`: the total amount of fee refunds that have been processed and sent to the recipient

Reviewer: "the total amount of fee refunds that have been received by the recipient"

> - `received`: the total amount of fee refunds that have been processed and sent to the recipient

Author: ```suggestion
- `received`: the total amount of fee refunds that have been received by the recipient
```

- `maxBlockNumber`: the highest block number for which fee refunds have been processed

> ### flashbots_getFeeRefundsByRecipient

Reviewer: Should we mention how often the data for these APIs is refreshed?

The `flashbots_getFeeRefundsByRecipient` JSON-RPC method returns detailed information about [fee refunds](/flashbots-protect/gas-fee-refunds) that have been earned by a specific address.  Our refund process calculates these values weekly.

```json
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "flashbots_getFeeRefundsByRecipient",
  "params": [
    {
      recipient,       // String, the address to query for fee refunds
      cursor,          // [optional] String, the cursor to continue from
    }
  ]
}
```

Responses are paginated and contain the following fields:

```json
{

>   "orders": [

Reviewer: Can we rename "orders" to "refunds"? To match the original spec

>   "orders": [

Author: ```suggestion
  "refunds": [
```

    {
      "hash": "0x...",
      "amount": "0x...",
      "blockNumber": "0x13ddaa4",
      "status": "pending",
      "recipient": "0x..."
    },
    ...
  ],
  "cursor": "0x..."
}
```

> The `"orders"` field contains an array of fee refund orders, each with the following fields:

Reviewer: Ditto on "orders" -> "refunds"

> The `"orders"` field contains an array of fee refund orders, each with the following fields:

Author: ```suggestion
The `"refund"` field contains an array of per-order fee refunds, each with the following fields:
```

> The `"refund"` field contains an array of per-order fee refunds, each with the following fields:

Reviewer: Tiny nit: the field is called "refunds" not "refund"

> The `"refund"` field contains an array of per-order fee refunds, each with the following fields:

Author: ```suggestion
The `"refunds"` field contains an array of per-order fee refunds, each with the following fields:
```
